src/components/TripList.jsx

```diff
@@ -0,0 +1,16 @@
+import { useContext } from "react";
+import TripCard from "./TripCard";
+import { TripContext } from "./TripContext";
+import styles from "./TripList.module.css";
+
+export default function TripList() {
+  const { trip } = useContext(TripContext);
+
+  return (
+    <div className={trip && styles["itinerary-container"]}>
```

Comment:
If my previous comment result in something, please do the same here.

Reply:
Thank you very much., unfortunately this way doesn´t work.